storage/connect/CMakeLists.txt

```diff
@@ -215,16 +228,21 @@ int main() {
     IF(ODBC_OK)
       INCLUDE_DIRECTORIES(${ODBC_INCLUDE_DIR})
       add_definitions(-DODBC_SUPPORT)
+      add_feature_info(CONNECT_ODBC "ON" "Support for ODBC in the CONNECT storage engine")
       SET(CONNECT_SOURCES ${CONNECT_SOURCES} tabodbc.cpp odbconn.cpp)
     ELSE()
       SET(ODBC_LIBRARY "")
+      add_feature_info(CONNECT_ODBC "OFF" "Support for ODBC in the CONNECT storage engine")
```

Comment:
single add_feature_info(CONNECT_ODBC  ODBC_OK "Support for ODBC in the CONNECT storage engine")
would suffice instead of using add_feature_info twice